lib/geo.js

```diff
@@ -10,7 +13,7 @@ exports.nearFilter = function nearFilter(where) {
   if (where && typeof where === 'object') {
     Object.keys(where).forEach(function (key) {
       var ex = where[key];
-
+      
```

Comment:
There is a white-space change here if I am reading this correctly. The original file contains an empty line, the new file contains a line with about 4 spaces. @cmoore4 could you PTAL?